pages/common/while.md

```diff
@@ -0,0 +1,11 @@
+# while
+
+> Simple shell loop
+
+- Read stdin and perform an action on every line.
+
+`while read line; do {{echo "$line"}}; done`
```

Comment:
FWIW, I wouldn't introduce a new syntax to differentiate between examples and tokens, I feel it would over-complicate things. I'm quite happy with either
- `while read line; do {{echo "$line"}}; done`
- `while read line; do echo "$line"; done`
- `while read line; do {{command using $line}}; done`  to be more generic